content/en/blog/_posts/2023-06-29-container-image-signature-verification/index.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: blog
+title: "Verifying container image signatures within CRI runtimes"
+date: 2023-06-29
+slug: container-image-signature-verification
+---
+
+**Author**: Sascha Grunert
+
+The Kubernetes community has been signing their container image-based artifacts
+since release v1.24. While the graduation of the [corresponding enhancement][kep]
+from `alpha` to `beta` in v1.26 introduced signatures for the binary artifacts,
+other projects followed the approach by providing image signatures for their
+releases, too. This means that they either create the signatures within their
+own CI/CD pipelines, for example by using GitHub actions, or rely on the
+Kubernetes [image promotion][promo] process to automatically sign the images by
+proposing pull requests to the [k/k8s.io][k8s.io] repository. A requirement for
+using this process is that the project is part of the `kubernetes` or
+`kubernetes-sigs` GitHub organization, so that they can utilize the community
+infrastructure for pushing images into staging buckets.
+
+[kep]: https://github.com/kubernetes/enhancements/issues/3031
+[promo]: https://github.com/kubernetes-sigs/promo-tools/blob/e2b96dd/docs/image-promotion.md
+[k8s.io]: https://github.com/kubernetes/k8s.io/tree/4b95cc2/k8s.gcr.io
+
+Assuming that a project now produces signed container image artifacts, how can
+one actually verify the signatures? It is possible to do it manually like
+outlined in the [official Kubernetes documentation][docs]. The problem with this
+approach is that it involves no automation at all and should be only done for
+testing purposes. In production environments, tools like the [sigstore
+policy-controller][policy-controller] can help with the automation. These tools
+provide a higher level API by using [Custom Resource Definitions (CRD)][crd] as
+well as an integrated [admission controller and webhook][admission] to verify
+the signatures.
+
+[docs]: /docs/tasks/administer-cluster/verify-signed-artifacts/#verifying-image-signatures
+[policy-controller]: https://docs.sigstore.dev/policy-controller/overview
+[crd]: /docs/concepts/extend-kubernetes/api-extension/custom-resources
+[admission]: /docs/reference/access-authn-authz/admission-controllers
+
+The general usage flow for an admission controller based verification is:
+
+![flow](flow.png "Admission controller flow")
```

Comment:
Nits:
1. Please use a figure shortcode
2. We prefer meaningful alt attributes; this helps with accessibility. Please consider fixing that.

eg see https://supercooldesign.co.uk/blog/how-to-write-good-alt-text